File: Algorithms/C++/Array/26_Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.empty()) {
            return 0;
        }

        int len = 0;
        for (int i = 1; i < nums.size(); i++) {
            if (nums[i] != nums[len]) {
                nums[++len] = nums[i];
            }
        }

        return len + 1;
    }
};

File: Algorithms/Swift/Array/26_Remove_Duplicates_from_Sorted_Array.swift
class Solution {
    func removeDuplicates(inout nums: [Int]) -> Int {
        if nums.isEmpty {
            return 0
        }

        var len = 0;
        for var i = 1; i < nums.count; i++ {
            if nums[i] != nums[len] {
                len += 1
                nums[len] = nums[i]
            }
        }

        return len + 1
    }
}
